src/openforms/emails/tests/test_send_mail_html.py

```diff
@@ -102,3 +104,72 @@ def test_oversize_content_raise_suspiciosu_operation(self):
                 "foo@sender.com",
                 ["foo@bar.baz"],
             )
+
+
+class DesignTokenFilterTest(TestCase):
+    """unit tests for ``emails.context._get_design_token_values``"""
+
+    def test_strip_px_unit(self):
+        self.design_tokens = {
+            "header-logo": {
+                "height": {"value": "100px"},
+                "width": {"value": "100px"},
+            }
+        }
+        result = _get_design_token_values(self.design_tokens)
+        self.assertEqual(result["logo"]["height"], "100px")
+        self.assertEqual(result["logo"]["width"], "100px")
+        self.assertEqual(result["logo"]["height_attr"], "100")
+        self.assertEqual(result["logo"]["width_attr"], "100")
+
+    def test_return_empty_string_for_non_px_unit(self):
+        self.design_tokens = {
+            "header-logo": {
+                "height": {"value": "100em"},
+                "width": {"value": "100em"},
+            }
+        }
+        result = _get_design_token_values(self.design_tokens)
+        self.assertEqual(result["logo"]["height"], "100em")
+        self.assertEqual(result["logo"]["width"], "100em")
+        self.assertEqual(result["logo"]["height_attr"], "")
+        self.assertEqual(result["logo"]["width_attr"], "")
+
+    def test_return_empty_string_for_auto_size(self):
+        """default for width: 'auto'"""
+        self.design_tokens = {"header-logo": {}}
+        result = _get_design_token_values(self.design_tokens)
+        self.assertEqual(result["logo"]["width"], "auto")
+        self.assertEqual(result["logo"]["width_attr"], "")
+
+
+class TemplateRenderingTest(TestCase):
+    """
+    integration test for the display of design token values
+
+    assumes that ``emails.context.get_wrapper_context`` works correctly,
+    hence the context variable is provided manually.
+    """
+
+    def test_design_token_values(self):
+        template = get_template("emails/wrapper.html")
+        ctx = {
+            "content": "<p>Hello there!</p>",
+            "main_website_url": "https://logoresizingdoneright.com",
+            "style": {
+                "logo": {
+                    "height": "150px",
+                    "width": "auto",
+                    "height_attr": "150",
+                    "width_attr": "",
+                }
+            },
+            "logo_url": "https://haarlemmermeer.open-formulieren.nl/media/logo/Haarlemmermeer-logo-2018-RGB-transparant_vsdibKW.png",
```

Comment:
please don't use/reference real URLs of real environments in publicly available code/repositories.

```suggestion
            "logo_url": "https://example.com/some-logo.png",
```